arm/Microsoft.Network/virtualWans/readme.md

```diff
@@ -34,6 +35,7 @@ This template deploys Virtual Wan
 | `roleAssignments` | array | [] | Complex structure, see below. | Optional. Array of role assignment objects that contain the 'roleDefinitionIdOrName' and 'principalId' to define RBAC role assignments on this resource. In the roleDefinitionIdOrName attribute, you can provide either the display name of the role definition, or it's fully qualified ID in the following format: '/providers/Microsoft.Authorization/roleDefinitions/c2f4ef07-c644-48eb-af81-4b1b4947fb11'
 | `tags` | object | {} | Complex structure, see below. | Optional. Tags of the Virtual Wan resource.
 | `cuaId` | string | "" | | Optional. Customer Usage Attribution id (GUID). This GUID must be previously registered.
+| `lockForDeletion` | bool | False |  | Optional. Switch to lock storage from deletion. |
```

Comment:
Should not reference storage, but virtualWan or so

Reply:
fixed